### PR TITLE
Some small changes to make codebuild checks work

### PR DESCRIPTION
**Pull request by kendase3**
Addresses OVER-7531

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - echo Build started on `date`
+      - make build-gondar test
```
Comment (nicholasbishop) [inline buildspec.yml]: Our Jenkins builder also builds the Windows version, and I think that's the part that Jenkins is actually choking on.

Comment (kendase3) [reply buildspec.yml]: this doesn't do that yet.  coming soon.

Comment (kendase3) [reply buildspec.yml]: this pr doesn't turn off the jenkins builders; they're already off.

**Comment by kendase3** [comment]
i'm now thinking i'm going to make a codebuild subdir to house these .yml files.  i think we'll have 4.

**Comment by kendase3** [comment]
deprecated by https://github.com/neverware/gondar/pull/278

**Pull request closed without merging**